services/cd-service/pkg/service/overview.go

```diff
@@ -108,6 +109,14 @@ func (o *OverviewServiceServer) getOverview(
 							app.QueuedVersion = *queuedVersion
 						}
 					}
+					app.UndeployVersion = false
+					if version != nil {
+						if release, err := s.GetApplicationRelease(appName, *version); err != nil && !errors.Is(err, os.ErrNotExist) {
+							return nil, err
+						} else if release != nil {
+							app.UndeployVersion = release.UndeployVersion
```

Comment:
does this mean that UndeployVersion would still be displayed in the UI like any other version? 
or it will just be returned with the `GetOverviewResponse`?

Reply:
for this PR it will be displayed like any other version, however in another change, I will add ui changes that make it more explicit